.changes/v3.6.0/786-features.md

```diff
@@ -0,0 +1 @@
+* `vcd_org_group` datasource allows to fetch an Organization group to use it with other resources [GH-786]
```

Comment:
Just taking your idea of capitalizing further:
```suggestion
* `vcd_org_group` datasource allows to fetch an Organization Group to use it with other resources [GH-786]
```

Reply:
Done